dotman-plugin/src/main/java/net/minevn/dotman/config/Discord.kt

```diff
@@ -1,28 +1,224 @@
-package net.minevn.dotman.config

```

Comment:
có vẻ như file này bị đổi line separator nên github hiện changes cả file
em tắt auto crlf trên máy rồi sửa lại như cũ (commit as-is) để khắc phục nha